### DISCUSSION TITLE
Device limit with MicroMDM

### DISCUSSION THREAD

**Discussion by honeyankit** [micromdm/micromdm#805] [Q&A]
Q1. Is there any device limit that can be enrolled in a single MircoMDM server? If yes, then what is the limit? If not, then is there any recommendation on how many devices can be enrolled into a single MicroMDM server?

**Comment by korylprince** [comment]
There is no hard limit to the number of devices MicroMDM can handle. It's going to depend on several factors - the performance of the hardware it's running on (cpu, disk speed, ram), how often you're sending commands to devices, and how you have MicroMDM configured (e.g. using in-memory command queue vs boltdb).

Very large companies use MicroMDM with  tens of thousands of devices on a single instance.
